pages/cheat_sheets/proteomics.md

```diff
@@ -14,66 +14,75 @@ dsw:
+  - [HUPO PSI Mass Spectrometry Controlled Vocabulary (PSI-MS CV)](https://doi.org/10.25504/FAIRsharing.sxh2dp)
+  - [NCBI Taxonomy (NCBITAXON)](https://doi.org/10.25504/FAIRsharing.fj07xj)
+  - [Experimental Factor Ontology (EFO)](https://doi.org/10.25504/FAIRsharing.1gr4tz)
 
 ### Sources for Reusable Data
-#### Source name 1
-- Description e.g. reference genome assembly
-- Standard License
-- Identifiers
-- How to access
+#### [Proteomecentral - ProteomeXchange](http://proteomecentral.proteomexchange.org/cgi/GetDataset)
+- Search across multiple repositories ([PRIDE](https://doi.org/10.25504/FAIRsharing.e1byny), [PeptideAtlas](https://doi.org/10.25504/FAIRsharing.dvyrsz), [MassIVE](https://doi.org/10.25504/FAIRsharing.LYsiMd), [JPOST](https://doi.org/10.25504/FAIRsharing.p899f7), [iProX](https://doi.org/10.25504/FAIRsharing.4Sj3vE) and [Panorama](https://doi.org/10.25504/FAIRsharing.uBpQ1q))
+- Usage license depended on respective repository
+- Identifiers:
+  - [Digital Object Identifier (DOI)](https://doi.org/10.25504/FAIRsharing.hFLKCn)
+  - PX Accession number
+
 
 ### Storage and Computing
-<!--Add information about e.g. NeLS-->
+<!--Add information about e.g. NeLS, update this section when SEEK is deployed for NOR-OS metadata?-->
+Proteomics data collected in Norway can be analyzed by the individual core facilities and stored in [Nels](https://nels.bioinfo.no/pages/user-terms.xhtml), [usegalaxy.no](https://usegalaxy.no) can be used to run [proteomics analysis workflows](https://training.galaxyproject.org/training-material/topics/proteomics/).  
 
-### Data Deposition Repository
 
-#### Repository Name 1
-- Standard License
-- Identifiers
-- How to submit data
-- Embargo
+### Data Deposition Repository
+#### [PRIDE](https://doi.org/10.25504/FAIRsharing.e1byny)
+- License: [Creative Commons Public Domain (CC0)](https://creativecommons.org/share-your-work/public-domain/cc0/)
+- Identifiers:
+  - [Digital Object Identifier (DOI)](https://doi.org/10.25504/FAIRsharing.hFLKCn)
+  - PX Accession number
+- [Submission guide](https://www.ebi.ac.uk/pride/markdownpage/submitdatapage)
+- [Embargo for up to publication](https://www.ebi.ac.uk/pride/markdownpage/submitdatapage#post-submission_steps)
 
-#### Repository Name 2
-- Standard License
-- Identifiers
-- How to submit data
-- Embargo
 
 ### Ethics and Regulations
 <!--Add information about laws and policies in Norway for relevant data types-->
-#### Data Type Name (e.g. Human Data)
-- Reidentifiable
-
-#### Data Type Name (e.g. Biodiversity Data)
-- Regulations
+#### Patient Data
+- Pre-approval for medicinal/health-related research projects from the [Regional Ethics Committee](https://rekportalen.no/#hjem/home) is required
+- Projects handling personal data are assessed at many institutions by [SIKT](https://sikt.no/fylle-ut-meldeskjema-personopplysninger)
+- Otherwise data must be fully anonymized - neither directly or indirectly identifiable to an individual
```

Comment:
"," missing after "otherwise". US spelling used in this line  (which might be fine depending on what we decide).